### PR TITLE
flesh out fixit participation guide

### PR DESCRIPTION
**Pull request by bsweger**
Starting to flesh out the participation guide. @melodykramer I left a placeholder for the prose.io doc. I also heard that you volunteered to write more, and I had some ideas when drafting this. Some of these could also be useful to incorporate into our NDoCH plans.

* Some kind of visual or doc on overall git/GitHub workflow as it pertains to written docs. Some of this is in the GitHub command-line blog that you and Greg wrote, but it might be helpful to show the workflow independent of the tool (command line vs GitHub client vs GitHub website)
* Speaking of the various git/GitHub editing tools, should we list the options somewhere? Or just pick one and make a recommendation for new users?
* You wrote a great GitHub prose.io guide. Do we also need a guide to editing docs via the GitHub website for projects that don't use prose?
* Do we need to extricate a stand-alone git explainer from the GitHub command-line blog so it's available to people who choose not to use the command line?

### CODE REVIEW

**Comment by bsweger** [comment]
@mbland pinging you on this too...should have done that yesterday

File: pages/docsprint/guides/participation.md [hunk 1]
```diff
@@ -4,4 +4,37 @@ permalink: /docsprint/guides/participation/
...
+Most 18F documentation is written in Markdown, a text-to-HTML conversion tool. Markdown is pretty straightforward.
+
+* Base [Markdown syntax](http://daringfireball.net/projects/markdown/).
+* For documentation hosted on GitHub, note that [GitHub-flavored markdown is in effect](https://help.github.com/articles/github-flavored-markdown/). It's very similar to regular Markdown but has some slight differences as well as some extra features like syntax highlighting.
+
+### Git and GitHub
+
+A large amount of 18F documentation is managed via git repositories. These are hosted on GitHub, which makes it easier to edit the Markdown documents. GitHub also provides tools for opening and tracking issues and submitting changes (called _pull requests_).
+
+If you're not familiar with Git and GitHub, these resources can get you started. You definitely don't need to be a git power user, and you can always ask questions in the #git Slack channel.
```
Comment (mbland) [inline pages/docsprint/guides/participation.md]: Should also #wg-documentation or #questions somewhere in here, probably.

File: pages/docsprint/guides/participation.md [hunk 2]
```diff
@@ -4,4 +4,37 @@ permalink: /docsprint/guides/participation/
...
 
-Here will be a guide to participating in a doc sprint.
+Taking our documentation to the next level is critical for scaling 18F. Here are some helpful materials and resources for participating in a doc sprint.
+
+## You Will Need
+To participate in an 18F doc sprint, you will need:
+
+* A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/) with [two-factor authentication enabled](https://help.github.com/articles/about-two-factor-authentication/)
+
+## Ways to Contribute
```
Comment (mbland) [inline pages/docsprint/guides/participation.md]: We need another section to highlight the "big three" bits of docs:

- READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
- Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
- Updating our [18F Guides](http://18f.github.io/guides/)

I also really like the more granular advice you give in this section. However you can think to marry the two would be wonderful.

File: pages/docsprint/guides/participation.md [hunk 3]
```diff
@@ -4,4 +4,62 @@ permalink: /docsprint/guides/participation/
...
 # Doc Sprint Participation Guide
 
-Here will be a guide to participating in a doc sprint.
+A well-organized doc sprint is designed to improve team documentation quickly and efficiently. Making sure 18F's documentation is accessible and comprehensive is critical for scaling what we do.  
+
+Documentation includes, but is not limited to:
+
+* READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
+* Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
+* Updating our [18F Guides](http://18f.github.io/guides/)
```
Comment (melodykramer) [inline pages/docsprint/guides/participation.md]: Should we specify that we're looking for design / content help on these guides? Where would that information go?

File: pages/docsprint/guides/participation.md [hunk 4]
```diff
@@ -4,4 +4,62 @@ permalink: /docsprint/guides/participation/
...
+
+**Here are some helpful materials and resources for participating in a doc sprint.**
+
+## <a name="todo"></a>Event Specific To-Do Lists
+
+Specific information and to-do list for each doc sprint:
+
+* [April 17 Onboarding Doc Sprint](../../onboarding/)
```
Comment (melodykramer) [inline pages/docsprint/guides/participation.md]: Are we going to link to all of the open issues? How will people find a list of tasks they can work on?

File: pages/docsprint/guides/participation.md (hunk 3 above)
Comment (mbland) [inline pages/docsprint/guides/participation.md]: There and https://github.com/18F/guides/issues.

**Comment by bsweger** [comment]
I might be overthinking this whole thing, but my original idea was to set this up so that we can set up a framework not only for this doc sprint but for future ones as well:

* participation.md = Universally applicable to every doc sprint (evergreen), with links to the event-specific doc sprint pages (right now, there's only one...onboarding.md)
* onboarding.md = the page that contains information specific to the onboarding doc sprint coming up on 4/17...so that's where we'd collect the issues that people should be working on

Does that make sense, or is it just going to end up confusing people? 

Either way, @melodykramer is right that we'll need to add specific links to what people can do on 4/17.

**Comment by bsweger** [comment]
pinging @mbland on this too

File: pages/docsprint/guides/participation.md (hunk 4 above)
Comment (melodykramer) [inline pages/docsprint/guides/participation.md]: It looks like there is now a template for guides and .io pages (cc: @elainekamlley) - what specifically are we asking the people to do? I am happy to flesh this section out.

Comment (mbland) [inline pages/docsprint/guides/participation.md]: @melodykramer Going to try to work with project/tech leads to figure out how they're collecting/burning down tasks, and to point them at the Sprint form to register activity.

Regarding the Guides, we're copying the template from the Agile guide. Most of the existing Guides are ported; just a few left. Plus, @nickbristow had a couple comments on accessibility to work in. ('Twould be nice to make the actual style files available as a--dare I say it?--_submodule_...) ;-)

**Comment by mbland** [comment]
@bsweger You plan sounds spot-on to me. Is there more you want to do to this PR, or is it not `[WIP]` anymore and ready to merge?

File: pages/docsprint/guides/participation.md (hunk 4 above)
Comment (melodykramer) [inline pages/docsprint/guides/participation.md]: Sounds good @mbland - thanks for the clarification! :+1:

**Comment by bsweger** [comment]
@mbland, @melodykramer and I feel there's more to be added, especially as teams start solidfying to-do lists, but let's go ahead with this initial version. thanks!

**Pull request merged**